### ISSUE TITLE
WARM REMINDER AGAIN!

### ISSUE THREAD

**Issue by huweibo** [kuandeng/LightGCN#6]
Unbelievably! There is a new bug in running GCMC and GCN model implemented by LightGCN.py. That is,  please specify the parameter "--adj_type"  to 'norm' or 'mean' (the default is 'pre') . Otherwise,  bug will find you!

**Comment by kuandeng** [comment]
I don't think this is a bug. You can modify the value of 'adj_type' in run command for different models. Thanks.

**Comment by huweibo** [comment]
Indeed，since there is no guide on how to run other models, I spent lots of time studing it. Thus, I just&nbsp;hope the authors can add the description to the README, Thanks.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"kuandeng"<notifications@github.com&gt;;
发送时间:&nbsp;2020年6月12日(星期五) 晚上11:04
收件人:&nbsp;"kuandeng/LightGCN"<LightGCN@noreply.github.com&gt;;
抄送:&nbsp;"weber"<1365615776@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [kuandeng/LightGCN] WARM REMINDER AGAIN! (#6)





 
I don't think this is a bug. You can modify the value of 'adj_type' in run command for different models. Thanks.

**Comment by hexiangnan** [comment]
In fact, we don't have the responsibility to maintain codes for
baselines... We will clean the codes..

On Fri, Jun 12, 2020 at 11:25 PM keven <notifications@github.com> wrote:
>
> Indeed，since there is no guide on how to run other models, I spent lots of time studing it. Thus, I just&nbsp;hope the authors can add the description to the README, Thanks.
>
>
>
>
> ------------------&nbsp;原始邮件&nbsp;------------------
> 发件人:&nbsp;"kuandeng"<notifications@github.com&gt;;
> 发送时间:&nbsp;2020年6月12日(星期五) 晚上11:04
> 收件人:&nbsp;"kuandeng/LightGCN"<LightGCN@noreply.github.com&gt;;
> 抄送:&nbsp;"weber"<1365615776@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
> 主题:&nbsp;Re: [kuandeng/LightGCN] WARM REMINDER AGAIN! (#6)
>
>
>
>
>
>
> I don't think this is a bug. You can modify the value of 'adj_type' in run command for different models. Thanks.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.



-- 
Xiangnan HE (Dr) :: Professor :: Vice Dean, School of Data Science ::
University of Science and Technology of China (USTC) :: 443 Huangshan
Road, Hefei, China 230027 :: (0551) 63607236 (DID) :: hexn@ustc.edu.cn
(E) :: http://staff.ustc.edu.cn/~hexn/ (W)
